### PR TITLE
Backstage weekly 20

### PR DESCRIPTION
**Pull request by dtuite**

### CODE REVIEW

File: content/blog/backstage-weekly-20-kubernetes-meetups/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Backstage Weekly 20 - Upcoming meetups and Kubernetes CRD support
+date: '2021-03-15T21:00:00.0Z'
+description: Heads up on two upcoming Backstage meetups and a roundup of the latest merged pull requests.
+tags: ['newsletter', 'kubernetes']
+---
+
+Believe it or not there are **two** Backstage meetups happening over the next 2 weeks! 📅
+
+The next official [Backstage Community Session](https://github.com/backstage/community/issues/7) will will take place on March 17th. As usual, you can suggest topics for the session and vote on your favorite topics using the linked GitHub issue. The current top suggestion is a presentation on how Expedia Group are using Backstage internally. Expedia is a huge organization with many engineers employed so this could be very interesting indeed.
```
Comment (padraigobrien) [inline content/blog/backstage-weekly-20-kubernetes-meetups/index.md:10]: two will's, "will will"

File: content/blog/backstage-weekly-20-kubernetes-meetups/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Bitrise plugin
+
+SDA SE contributed a plugin for mobile app CI/CD platform [Bitrise](https://www.bitrise.io/) this week. It can quickly show you when your mobile builds are failing and even allow you to download the artefact generated by the builds. [#4746](https://github.com/backstage/backstage/pull/4746)
+
+![a list of builds inside Bitrise showing the date it ran, the success status and a link to download the artefact. Some portions are blurred out.](./bitrise-plugin.png)
+
+### Backstage errors package
+
+The Backstage team have started an effort to standardize error reporting in Backstage in order to improve consistency when things go wrong. There is now a `@backstage/errors` package and an error panel which showsT the most pertinent information to the user immediately and can also expand to show more details. [#4903](https://github.com/backstage/backstage/pull/4903)
```
Comment (padraigobrien) [inline content/blog/backstage-weekly-20-kubernetes-meetups/index.md:34]: showsT, possible spelling mistake

**Review by padraigobrien — CHANGES_REQUESTED** [review]

File: content/blog/backstage-weekly-20-kubernetes-meetups/index.md (hunk 2 above)
Comment (Xantier) [inline content/blog/backstage-weekly-20-kubernetes-meetups/index.md:34]:
```suggestion
The Backstage team have started an effort to standardize error reporting in Backstage in order to improve consistency when things go wrong. There is now a `@backstage/errors` package and an error panel which shows the most pertinent information to the user immediately and can also expand to show more details. [#4903](https://github.com/backstage/backstage/pull/4903)
```

File: content/blog/backstage-weekly-20-kubernetes-meetups/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+I've seen a number of discussions about when and how to display errors to the user in Backstage so it's good to see standardization begin in this area.
```
Comment (Xantier) [inline content/blog/backstage-weekly-20-kubernetes-meetups/index.md:36]: 'Standardization' here is american spelling, whereas 'artefact' above is from them Brits.

**Review by Xantier — APPROVED** [review]

File: content/blog/backstage-weekly-20-kubernetes-meetups/index.md (hunk 3 above)
Comment (dtuite) [reply content/blog/backstage-weekly-20-kubernetes-meetups/index.md:36]: They actually have different meaning according to some people. https://english.stackexchange.com/a/37976

Anyway, no big deal here.

**Pull request merged**